content/blog/2019-08-13-popler.md

```diff
@@ -0,0 +1,81 @@
+---
+slug: "popler"
+title: Synthesizing population time-series data from the USA Long Term Ecological Research Network
+package_version: 0.2.0
+authors:
+  - Aldo Compagnoni
+date: 2019-08-13
+categories: blog
+topicid:
+tags:
+- Software Peer Review
+- R
+- peer-reviewed
+- popler
+- open long-term population data
+- US Long Term Ecological Research Network data
+- LTER
+- online database
+- database structure
+- PostgreSQL
+- API
+- R package
+- data synthesis
+- comparative analysis
+---
+
+
+### Introduction
+
+The availability of large quantities of freely available data is revolutionizing the world of ecological research. Open data maximizes the opportunities to perform comparative analyses and meta-analyses. Such synthesis efforts will increasingly exploit "population data", which we define as time series of population abundance. Such population data plays a central role in testing ecological theory and guiding management decisions. One of the richest sources of open access population data is the [USA Long Term Ecological Research (LTER) Network](https://lternet.edu/). However, LTER data presents the drawback common to all ecological time-series: extreme heterogeneity derived from differences in sampling designs. 
+We experienced this heterogeneity first hand, upon embarking in our own comparative analysis of population data. Specifically, we noticed that heterogeneities in sampling design made datasets  hard to compare, and therefore hard to search and analyze.
+
+To alleviate these issues, we created `popler`: an online PostgreSQL database (henceforth "popler online database"), and associated R client (henceforth "popler R package"). `popler` accommodates raw population time-series data using the same structure for all datasets. Using raw data, without aggregation, promotes flexibility in data analysis. Moreover, using a common data structure implies that all datasets share the same variables, or the same type of variables. As a result, popler facilitates comparing, manipulating, and retrieving population data. 
```

Comment:
For For formatting of package names, functions, and code, follow the [tidyverse style guidance](https://style.tidyverse.org/documentation.html#r-code), so don't use code font (backticks) for package names.